src/expr/type.h

```diff
@@ -373,6 +373,12 @@ class CVC4_PUBLIC Type {
    */
   bool isSet() const;
 
+  /**
+   * Is this a Bag type?
+   * @return true if the type is a Bag type
+   */
+  bool isBag() const;
```

Comment:
You can remove this.

Reply:
Done. removed from header and source

Reply:
`bool Sort::isBag() const { return TypeNode::fromType(*d_type).isBag(); }`